packages/grafana-ui/src/components/Carousel/Carousel.tsx

```diff
@@ -88,72 +98,87 @@ export const Carousel: React.FC<CarouselProps> = ({ images }) => {
   }
 
   return (
-    <div onKeyDown={handleKeyDown} tabIndex={0}>
+    <>
       <div className={cx(styles.imageGrid)}>
         {validImages.map((image, index) => (
-          <div key={image.path} onClick={() => openPreview(index)} style={{ cursor: 'pointer' }}>
+          <button
```

Comment:
I'd add `type='button'` just in case, never know where it's gonna end up being used.